webapp/src/ts/modules/messages/messages.routes.ts

```diff
@@ -0,0 +1,13 @@
+import { Routes } from '@angular/router';
+
+import { RouteGuardProvider } from '@mm-providers/route-guard.provider';
+import { MessagesComponent } from './messages.component';
+
+export const routes: Routes = [
+    {
+        path: 'messages',
+        component: MessagesComponent,
+        data: { permissions: [ 'can_view_messages' ] },
+        canActivate: [ RouteGuardProvider ],
+    },
+];
```

Comment:
Indentation again.

Reply:
Thanks! My IDE isn't picking up the eslint for some reason. Fixed